src/Flux/layers.jl

```diff
@@ -39,14 +41,15 @@ function diffeq_adjoint(p,prob,args...;u0=prob.u0,kwargs...)
 end
 
 diffeq_adjoint(p::TrackedVector,args...;kwargs...) = Flux.Tracker.track(diffeq_adjoint, p, args...; kwargs...)
-Flux.Tracker.@grad function diffeq_adjoint(p::TrackedVector,prob,args...;u0=prob.u0,kwargs...)
+@grad function diffeq_adjoint(p::TrackedVector,prob,args...;
+                              u0=prob.u0,sensealg=SensitivityAlg(backsolve=true),kwargs...)
```

Comment:
we really shouldn't default to something that has a wide range of methods where it fails